Readme.md

```diff
@@ -99,6 +108,14 @@ The `config.json` looks like the following:
             "httpBody": {
                 "foo": [1, 2, 3]
             }
+        },
+        "slackHook": {
+            "reqUrl": "https://hooks.slack.com/services/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w",
```

Comment:
Just for safety: This is not a real, functioning token, right?

Reply:
Thanks for asking, it's not a real token.